README.md

```diff
@@ -1,5 +1,10 @@
 # WakaTime CLI
 
+| branch  | build | coverage |
```

Comment:
The coverage of `develop` branch is mainly for internal purposes and I guess we don't need to have it on top of the README. Let's keep it clean. Badge for master would be enough here I guess.

Reply:
As we want to `develop` branch for testing and alpha releases I believe having it at the top would be useful for people who want to make sure build is passing. If we add it why not to also have its coverage reported and exposed?